package.json

```diff
@@ -11,12 +11,23 @@
     "type": "git",
     "url": "http://github.com/thoughtbot/Abstraction"
   },
+  "keywords": [
+    "sass",
+    "bourbon",
+    "neat",
+    "gulp",
+    "ghost",
+    "theme"
+  ],
   "devDependencies": {
-    "gulp": "~3.8.0",
-    "gulp-sass": "~1.3.2",
-    "gulp-jshint": "~1.9.0",
-    "gulp-autoprefixer": "~2.1.0",
-    "gulp-notify": "~2.0.0",
-    "jshint-stylish": "~1.0.0"
-  }
+    "bourbon": "^5.0.0-beta.8",
+    "bourbon-neat": "^2.1.0",
+    "gulp": "^3.9.1",
+    "gulp-autoprefixer": "^4.0.0",
+    "gulp-jshint": "^2.0.4",
+    "gulp-sass": "^3.1.0",
+    "jshint": "^2.9.5",
+    "jshint-stylish": "^2.2.1"
+  },
+  "license": "ISC"
```

Comment:
Since we're adding this, should we add a `LICENSE.md` file to the repo, as well?